src/__experimental__/components/input-icon-toggle/input-icon-toggle.style.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+import Colors from 'style/themes/base';
+
+const { colors } = Colors;
+const borderColor = '#668592';
+
+const Label = styled.label`
+  background-color: ${props => (props.isHovered ? colors.tertiary : colors.primary)};
+  color: ${props => (props.isHovered ? colors.white : colors.text.body)};
+  border-left: 1px solid ${borderColor};
+  cursor: pointer;
+  padding: 6px;
+  margin: -2px -6px; /*correction for parent element padding*/
+`;
+
+export default Label;
```

Comment:
After you declare the styles, you should add default props, and set defaultProps.theme to the baseTheme.

Reply:
good point, fixed